docs/run_test/platform.rst

```diff
@@ -294,3 +296,43 @@ be deployed to the same configured region.
    Update the default user name for the AMI you use to launch the instance.
    For an Ubuntu AMI, the user name is ubuntu. Please refer to the
    `general prerequisites for connecting to the instance <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/connection-prereqs.html>`_.
+
+Run on WSL
+------------
+
+WSL is supported cross all platforms by the guest layer in a node. So, it can be
+run with Local, Ready, Azure, AWS, BareMetal, etc. LISA supports,
+
+* Provisioning WSL from a clean environment, or reuse existing WSl environment.
```

Comment:
Provisioning WSL from a clean environment, or reuse existing WSL environment.

Reply:
Good catch. You can use suggested change to highlight the diff parts.